### ISSUE TITLE
PR #461 requires GLIBC 2.27 on x86_64 and i686 linux build

### ISSUE THREAD

**Issue by mxaddict** [navcoin/navcoin-core#482]
PR #461 requires GLIBC 2.27 on x86_64 and i686 linux build

We will need to add back compat code for symbols used to maintain compatibility for next navcoin release

We should use:
```
'80386':  GLIBC (2,1),
'X86-64': GLIBC (2,2,5),
```

**Comment by mxaddict** [comment]
Will work on this after #461 is merged into master
